builder/yandex/builder.go

```diff
@@ -31,7 +32,19 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, []string, error) {
 	if errs != nil {
 		return nil, warnings, errs
 	}
-	return nil, warnings, nil
+	generatedData := []string{
```

Comment:
To inform users of available build template variables we've started to document the variables on the builder specific page under the `Build template data` section. Here is a link to an Amazon example https://www.packer.io/docs/builders/amazon/ebs#build-template-data, the source file is [here](https://github.com/hashicorp/packer/blob/master/website/pages/docs/builders/amazon/ebs.mdx#build-shared-information-variables)

Can you add this section to the Yandex builder docs please.

Reply:
Hi @nywilken !
Thank you for feedback. Bring some update to Yandex builder [doc](website/pages/docs/builders/yandex.mdx).